great_tables/_gt_data.py

```diff
@@ -869,8 +870,7 @@ class FootnoteInfo:
 
 @dataclass(frozen=True)
 class StyleInfo:
-    locname: str
-    locnum: int
+    locname: Loc
```

Comment:
locname with a Loc class now fully identifies a style info, so no need for locnum

Reply:
Great!